File: app/src/main/java/za/co/gundula/app/arereyeng/model/Journey.java
package za.co.gundula.app.arereyeng.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by kgundula on 2017/02/26.
 */

public class Journey {

    @SerializedName("geometry")
    @Expose
    private Geometry geometry;
    @SerializedName("time")
    @Expose
    private String time;
    @SerializedName("timeType")
    @Expose
    private String timeType;
    @SerializedName("fareProducts")
    @Expose
    private List<String> fareProducts = null;

    public Journey() {
    }

    public Journey(Geometry geometry, String time, String timeType, List<String> fareProducts) {
        this.geometry = geometry;
        this.time = time;
        this.timeType = timeType;
        this.fareProducts = fareProducts;
    }

    public Geometry getGeometry() {
        return geometry;
    }

    public void setGeometry(Geometry geometry) {
        this.geometry = geometry;
    }

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }

    public String getTimeType() {
        return timeType;
    }

    public void setTimeType(String timeType) {
        this.timeType = timeType;
    }

    public List<String> getFareProducts() {
        return fareProducts;
    }

    public void setFareProducts(List<String> fareProducts) {
        this.fareProducts = fareProducts;
    }


}

File: app/src/main/java/za/co/gundula/app/arereyeng/model/Journey.kt
package za.co.gundula.app.arereyeng.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by kgundula on 2017/02/26.
 */
class Journey(@SerializedName("geometry")
              @Expose var geometry: Geometry?, @SerializedName("time")
              @Expose var time: String?, @SerializedName("timeType")
              @Expose var timeType: String?, @SerializedName("fareProducts")
              @Expose var fareProducts: List<String>?) {

}